staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1359,6 +1366,13 @@ message ContainerStatus {
     string image_id = 17;
     // User identities initially attached to the container
     ContainerUser user = 18;
+
+    // Returns the stop signal used by the container runtime to terminate the container 
+    ContainerLifecycleStatus lifecycle = 19;
```

Comment:
We could directly add the stop signal. I am not seeing a strong need to nest under Lifecycle message here.